aten/src/ATen/cuda/jiterator.cu

```diff
@@ -308,38 +311,47 @@ void jitted_gpu_kernel_dynamic(
     }
   }
 
-  jitted_gpu_kernel_dynamic_impl(kernel_name, iter, f, needs_dynamic_casting, extra_args);
+  jitted_gpu_kernel_dynamic_impl(kernel_name, iter, f, needs_dynamic_casting, extra_args, return_by_ref);
 }
 
 } // namespace native
 
 namespace cuda {
 
-at::Tensor CompileAndLaunchKernel(
+std::vector<at::Tensor> CompileAndLaunchKernel(
```

Comment:
you probably want SmallVector here to reduce overheads (and for input tensors also)

Reply:
done.